src/ssl.c

```diff
@@ -36061,6 +36061,9 @@ static int hash2mgf(enum wc_HashType hType)
     case WC_HASH_TYPE_SHA3_512:
     case WC_HASH_TYPE_BLAKE2B:
     case WC_HASH_TYPE_BLAKE2S:
+#ifndef WOLFSSL_SHA224
+    case WC_HASH_TYPE_SHA224:
```

Comment:
We already have a WC_HASH_TYPE_SHA224 in this switch statement, above.  Would it be better to just move the "#ifdef WOLFSSL_SHA224" at that point inside the "case WC_HASH_TYPE_SHA224:" to avoid duplicate cases listed?

Reply:
Merged it into existing #ifdef WOLFSSL_SHA224.